content/using-atom/sections/atom-packages.md

```diff
@@ -62,26 +66,26 @@ For example `apm install emmet@0.1.5` installs the `0.1.5` release of the [Emmet
 
 You can also use `apm` to find new packages to install. If you run `apm search`, you can search the package registry for a search term.
 
-```
+``` command-line
 $ apm search coffee
-Search Results For 'coffee' (5)
-├── coffee-trace Add smart trace statements to coffee files with one keypress each. (77 downloads, 3 stars)
-├── coffee-navigator Code navigation panel for Coffee Script (557 downloads, 8 stars)
-├── atom-compile-coffee This Atom.io Package compiles .coffee Files on save to .js files. (myJavascript.coffee -> myJavascript.js) (349 downloads, 4 stars)
-├── coffee-lint CoffeeScript linter (3336 downloads, 18 stars)
-└── git-grep `git grep` in atom editor (1224 downloads, 9 stars)
+> Search Results For 'coffee' (5)
+> ├── coffee-trace Add smart trace statements to coffee files with one keypress each. (77 downloads, 3 stars)
+> ├── coffee-navigator Code navigation panel for Coffee Script (557 downloads, 8 stars)
+> ├── atom-compile-coffee This Atom.io Package compiles .coffee Files on save to .js files. (myJavascript.coffee -> myJavascript.js) (349 downloads, 4 stars)
+> ├── coffee-lint CoffeeScript linter (3336 downloads, 18 stars)
+> └── git-grep `git grep` in atom editor (1224 downloads, 9 stars)
 ```
 
 You can use `apm view` to see more information about a specific package.
 
-```
+``` command-line
 $ apm view git-grep
-git-grep
-├── 0.7.0
-├── git://github.com/mizchi/atom-git-grep
-├── `git grep` in atom editor
-├── 1224 downloads
-└── 9 stars
-
-Run `apm install git-grep` to install this package.
+> git-grep
+> ├── 0.7.0
+> ├── git://github.com/mizchi/atom-git-grep
+> ├── `git grep` in atom editor
+> ├── 1224 downloads
+> └── 9 stars>
```

Comment:
That's intentional:

https://github.com/gjtorikian/extended-markdown-filter#command-line-highlighting